forum/qa-plugin/q2a-badges-master/qa-badge-check.php

```diff
@@ -358,20 +358,17 @@ function answer_vote_down($event,$event_user,$params) {
 			if($event_user) $this->check_voter($event_user);
 		}
 
-		function check_voted($uid) {
-						// upvotes received
+                function check_voted($uid) {
+                    // upvotes received
 			
-			$votes = qa_db_read_one_assoc(
-				qa_db_query_sub(
-					'SELECT upvoteds FROM ^userpoints WHERE userid=#',
-					$uid
-				),
-				true
-			);
-			$badges = array('liked','loved','revered');
+                    $votesArray = qa_db_query_raw('SELECT upvoteds FROM qa_userpoints WHERE userid='.$uid.' LIMIT 1');
+                    foreach($votesArray as $vote) {

```

Comment:
PSR-2 (brak spacji)

Reply:
W którym miejscu?

Reply:
Standard bardzo precyzyjnie określa wygląd foreach: https://www.php-fig.org/psr/psr-2/#55-foreach i zwraca uwagę, aby zwrócić uwagę na ułożenie nawiasów, spacji i klamer.

Tak, wiem że to wygląda jak czepianie się na siłę, ale gdybyśmy mieli jakiś linter pod PSR-2 to również by tego nie przepuścił - ba, nie przepuściłby żadnego najmniejszego błędu.